### PR TITLE
Event Dashboard: Logos and Columns scaled properly #3091   

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**
Column size adjusted and link to the logo is added.
Please review.

### CODE REVIEW

**Comment by Himanshi-Khandelwal** [comment]
Here is the pic:
screenshot from 2017-02-28 14 10 13

**Comment by Himanshi-Khandelwal** [comment]
@niranjan94 which column or which part is giving the problem?

**Comment by niranjan94** [comment]
@Himanshi-Khandelwal the logo column. The logo's don't maintain their aspect ratio since your forcing both height and width.

**Comment by niranjan94** [comment]
As for the file conflicts, Github is clearly saying which files have a conflict. Please resolve the conflicts.

**Comment by Himanshi-Khandelwal** [comment]
now it is looking like this:
screenshot from 2017-03-02 23 31 15

**Comment by Himanshi-Khandelwal** [comment]
Done changes. Also to resolve the conflict , I have done the changes online. Hence, another commit message is being shown ( 1 through terminal + 1 done online) commit.

**Comment by niranjan94** [comment]
@Himanshi-Khandelwal doesn't matter where the commit was done ... Please squash them all into one commit.

File: app/templates/gentelella/users/events/_table.html [hunk 1]
```diff
@@ -42,7 +42,11 @@
                 {% endfor %}
 
                 <tr>
+
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/_table.html]: Unnecessary new line added here.

File: app/templates/gentelella/users/events/_table.html [hunk 2]
```diff
@@ -42,7 +42,11 @@
...
+                    <td><a href="{{ url_for('events.details_view', event_id=event.id) }}">  <img src="{{ event.logo }}"  onerror="imgError(this); " style=" width:70px;"><p style ="padding:8px;">{{ event.name }}</p></a></td>
+
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/_table.html]: Unnecessary new line added here.

File: app/templates/gentelella/users/events/_table.html [hunk 3]
```diff
@@ -42,7 +42,11 @@
...
                     <td>  <img src="{{ event.logo }}"  onerror="imgError(this); " style="height:60px ; width:60px;"><a href="{{ url_for('events.details_view', event_id=event.id) }}"><br>&ensp;{{ event.name }}</a></td>
+
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/_table.html]: Unnecessary new line added here.

File: app/templates/gentelella/users/events/index.html [hunk 4]
```diff
@@ -93,12 +93,21 @@
     <script type="text/javascript">
         $('.table').DataTable({
         "columnDefs": [
+
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/index.html]: Unnecessary new line added here.

File: app/templates/gentelella/users/events/index.html [hunk 5]
```diff
@@ -93,12 +93,21 @@
...
+	         { "width" : "3%", "targets" : 0 },
+	         { "width" : "20%", "targets": 1 },
+            { "width" : "28%", "targets": 2 },
+	         { "width" : "15%", "targets": 3 },
+	         { "width" : "10%", "targets": 4 },
+            { "width" : "20%", "targets": 5 }
+
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/index.html]: Unnecessary new line added here.

File: app/templates/gentelella/users/events/index.html [hunk 6]
```diff
@@ -93,12 +93,21 @@
...
 	         { "width" : "8%", "targets" : 0 },
 	         { "width" : "15%", "targets": 1 },
             { "width" : "23%", "targets": 2 },
 	         { "width" : "10%", "targets": 3 },
 	         { "width" : "10%", "targets": 4 },
 	         { "width" : "10%", "targets": 6 }
+
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/index.html]: Unnecessary new line added here.

File: app/templates/gentelella/users/events/index.html [hunk 7]
```diff
@@ -93,12 +93,21 @@
     <script type="text/javascript">
         $('.table').DataTable({
         "columnDefs": [
+
+	         { "width" : "3%", "targets" : 0 },
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/index.html]: Duplication. Which one is the right one ?

**Comment by Himanshi-Khandelwal** [comment]
@niranjan94 in my texteditor, it is not showing any line gaps in between

**Comment by niranjan94** [comment]
@Himanshi-Khandelwal as I have said before, Please don't close and re-open new PRs like this. It's very hard to keep track of conversations. Just force-update the same branch and your PR will get automatically updated.

**Pull request closed without merging**